### PR TITLE
Allow skip back and forwards between rounds

### LINKED ISSUE #684: Jump by round in history as well as by turn
At the moment, we have two options:

- |< , >| : jump to beginning/end
- < , > : jump by action

Would it be possible to add an intermediate level (<< , >> maybe?), that allows jumping by round? This would allow reviewing a game quickly, but still fine-grained enough to be able to track how the game changed.

### PR DESCRIPTION
**Pull request by jenf**
fixes #684

### CODE REVIEW

File: assets/app/view/game/actionable.rb [hunk 1]
```diff
@@ -68,6 +69,7 @@ def process_action(action)
 
         store(:tile_selector, nil, skip: true)
         store(:game, game)
+        store(:fullgame, game)
```
Comment (tobymao) [inline assets/app/view/game/actionable.rb:72]: double render

**Comment by tobymao** [comment]
i'm not a fan of this approach. i think we should instead store outside like round_history, storing 2 game objects seems very heavy weight. either that or in the game_data we could store pointers to round history

something like round -> action .id maybe.

File: assets/app/view/game/actionable.rb (hunk 1 above)
Comment (jenf) [reply assets/app/view/game/actionable.rb:72]: ahh, i'd assumed it would swallow the second render

**Comment by jenf** [comment]
Yeah, we can probably just store the round history list, but we'd still have to process the full game if your in history mode.

**Comment by tobymao** [comment]
don't we always process the full game

**Comment by jenf** [comment]
Not if you come in via an URL argument (e.g. http://localhost:9292/hotseat/hs_qnhsajwz_2284?action=449)
But I can cover that case.

**Comment by tobymao** [comment]
why does that change?

**Comment by jenf** [comment]
The game object only loads the first n actions, so you cannot go forward again

**Comment by tobymao** [comment]
that's fine

**Comment by jenf** [comment]
so, just for clarity, store the round history list in a needs?

**Pull request merged**